From 7cf395ffbb84cc3a9a1417c0bd3d55e56db6322c Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Wed, 11 May 2016 14:47:18 +0800
Subject: [PATCH 1060/5242] MLK-12792 ARM: dts: imx6ull-14x14-ddr3-arm2:
 improve enet clock timing

commit  41fd2bcf4a16b764e67cb852cea3251e13ec4d29 from
https://source.codeaurora.org/external/imx/linux-imx.git

Improve enet data/txc clock timing suggested by HW team.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 45a4e0e..24292e2 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -480,18 +480,18 @@
 				MX6UL_PAD_ENET1_TX_EN__ENET1_TX_EN	0x1b0b0
 				MX6UL_PAD_ENET1_TX_DATA0__ENET1_TDATA00	0x1b0b0
 				MX6UL_PAD_ENET1_TX_DATA1__ENET1_TDATA01	0x1b0b0
-				MX6UL_PAD_ENET1_TX_CLK__ENET1_REF_CLK1	0x4001b0a8
+				MX6UL_PAD_ENET1_TX_CLK__ENET1_REF_CLK1	0x4001b0a0
 			>;
 		};
 
 		pinctrl_enet2: enet2grp {
 			fsl,pins = <
-				MX6UL_PAD_GPIO1_IO07__ENET2_MDC		0x1b0b0
+				MX6UL_PAD_GPIO1_IO07__ENET2_MDC		0x1b098
 				MX6UL_PAD_GPIO1_IO06__ENET2_MDIO	0x1b0b0
-				MX6UL_PAD_ENET2_TX_DATA0__ENET2_TDATA00	0x1b0b0
-				MX6UL_PAD_ENET2_TX_DATA1__ENET2_TDATA01	0x1b0b0
-				MX6UL_PAD_UART4_TX_DATA__ENET2_TDATA02	0x1b0b0
-				MX6UL_PAD_UART4_RX_DATA__ENET2_TDATA03	0x1b0b0
+				MX6UL_PAD_ENET2_TX_DATA0__ENET2_TDATA00	0x1b0a0
+				MX6UL_PAD_ENET2_TX_DATA1__ENET2_TDATA01	0x1b0a0
+				MX6UL_PAD_UART4_TX_DATA__ENET2_TDATA02	0x1b0a0
+				MX6UL_PAD_UART4_RX_DATA__ENET2_TDATA03	0x1b0a0
 				MX6UL_PAD_ENET2_TX_CLK__ENET2_TX_CLK	0x4001b0a8
 				MX6UL_PAD_ENET2_TX_EN__ENET2_TX_EN	0x1b0b0
 				MX6UL_PAD_ENET2_RX_DATA0__ENET2_RDATA00	0x1b0b0
-- 
1.7.9.5

